google/cloud/alloydb/connector/instance.py

```diff
@@ -83,6 +82,25 @@ def __init__(
         self._current: asyncio.Task = self._schedule_refresh(0)
         self._next: asyncio.Task = self._current
 
+    def _parse_instance_uri(self, instance_uri: str) -> Tuple[str, str, str, str, str]:
+        # should take form "projects/<PROJECT>/locations/<REGION>/clusters/<CLUSTER>/instances/<INSTANCE>"
+        if INSTANCE_URI_REGEX.fullmatch(instance_uri) is None:
+            raise ValueError(
+                "Arg `instance_uri` must have "
+                "format: projects/<PROJECT>/locations/<REGION>/clusters/<CLUSTER>/instances/<INSTANCE>, "
+                f"got {instance_uri}."
+            )
+
+        instance_uri_split = INSTANCE_URI_REGEX.split(instance_uri)
+
+        return (
+            instance_uri,
```

Comment:
it is not necessary to return `instance_uri` it is an input to the function which means you already have access to it wherever the function is being called.
```suggestion
```

Reply:
I have removed the extra return for instance_uri. Although I am assigning it to a class member variable from argument to store it. Is it fine?